playbooks/roles/matomo/templates/matomo_archive_reports.sh.j2

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker exec -u www-data matomo php /var/www/html/console core:archive --url=https://{{ MATOMO_APP_DOMAIN}} > /dev/null
```

Comment:
Can you please add some documentation to this script, just a point on what it does and why it's necessary.

It's easy to forget all about this when we stop touching this instance.

Reply:
@viadanna, done. :+1: